internal/services/newrelic/new_relic_monitor_resource.go

```diff
@@ -282,7 +305,21 @@ func (r NewRelicMonitorResource) Create() sdk.ResourceFunc {
 				return fmt.Errorf("creating %s: %+v", id, err)
 			}
 
+			if len(model.MonitoredSubscription) > 0 {
+				monitoredSubscriptionsProperties := monitoredsubscriptions.MonitoredSubscriptionProperties{
+					Properties: &monitoredsubscriptions.SubscriptionList{
+						MonitoredSubscriptionList: expandMonitorSubscriptionList(model.MonitoredSubscription),
+					},
+				}
+
+				monitorId := monitoredsubscriptions.NewMonitorID(id.SubscriptionId, id.ResourceGroupName, id.MonitorName)
+				if err := metadata.Client.NewRelic.MonitoredSubscriptionsClient.CreateOrUpdateThenPoll(ctx, monitorId, monitoredSubscriptionsProperties); err != nil {
+					return fmt.Errorf("creating NewRelic Monitored Subscriptions of %s: %+v", id, err)
```

Comment:
There should be a space between `NewRelic`, can you please update the other error messages as well?
```suggestion
					return fmt.Errorf("creating New Relic Monitored Subscriptions of %s: %+v", id, err)
```

Reply:
fixed